NuKeeper.Tests/RepositoryInspection/PackagesFileReaderTests.cs

```diff
@@ -69,8 +70,14 @@ public void TwoPackagesShouldBeRead()
 
             Assert.That(packages, Is.Not.Null);
             Assert.That(packages.Count, Is.EqualTo(2));
+
+            PackageAssert.IsPopulated(packages[0]);
             Assert.That(packages[0].Id, Is.EqualTo("foo"));
+            Assert.That(packages[0].Version, Is.EqualTo(new NuGetVersion("1.2.3.4")));
+
+            PackageAssert.IsPopulated(packages[1]);
             Assert.That(packages[1].Id, Is.EqualTo("bar"));
+            Assert.That(packages[1].Version, Is.EqualTo(new NuGetVersion("2.3.4.5")));
```

Comment:
There's a good discussion here https://softwareengineering.stackexchange.com/questions/7823/is-it-ok-to-have-multiple-asserts-in-a-single-unit-test

In particular:
> Tests should fail for one reason only, but that doesn't always mean that there should be only one Assert statement. IMHO it is more important to hold to the "Arrange, Act, Assert" pattern.


This test does seem fairly solid in that it's not going to fail for many different reasons. Maybe the test is too big? Maybe the function behind needs breaking down?

16 asserts makes me nervous - has to be a code smell...